### ISSUE TITLE
last release is breaking my CI 

### ISSUE THREAD

**Issue by 12rambau** [mapado/haversine#54]
I'm using haversine for a while now and I really like it's simplicity. 
My today's latest build failed because you introduced something new in the lib:
https://github.com/12rambau/sepal_ui/runs/7278765322?check_suite_focus=true

```
def_ensure_lat_lon(lat: float, lon: float):
"""
    Ensure that the given latitude and longitude have proper values. An exception is raised if they are not.
    """
if lat < -90or lat > 90:
>           raiseValueError(f"Latitude {lat} is out of range [-90, 90]")
```

it should be `<=` and `>=` right ?

**Comment by jdeniau** [comment]
Hi @12rambau,

you are totally right! I will push a fix right now.

You CI will probably continue to fail though as your latitude seems to be -180, and latitude should be between `-90` and `90`

**Comment by jdeniau** [comment]
OK I was about to respond the same thing.
The code seems correct, but your latitude is -180, where latitude should be between `[-90, 90]`

> The Equator has a latitude of 0°, the [North Pole](https://en.wikipedia.org/wiki/North_Pole) has a latitude of 90° North (written 90° N or +90°), and the [South Pole](https://en.wikipedia.org/wiki/South_Pole) has a latitude of 90° South (written 90° S or −90°)
[wikipedia](https://en.wikipedia.org/wiki/Latitude#The_graticule_on_the_sphere)

**Comment by 12rambau** [comment]
that's it I checked my code and effectively when nothing is added I'm using the full map zoom level in the Google Earth Engine convention which is lng/lat... 

So nice choice to add this check in the lib ;-) I'll correct the bug from my lib and close this issue once it works

**Comment by 12rambau** [comment]
yep and as it was not tested before, my CI was going full straight ahead without complaining

**Comment by jdeniau** [comment]
No problem, it's nice to have comment about haversine usage !
